packages/data-service/src/connect-mongo-client.ts

```diff
@@ -99,6 +99,7 @@ function connectionOptionsToMongoClientParams(
 
   const options: MongoClientOptions = {
     monitorCommands: true,
+    tlsCertificateFile: connectionOptions.tlsCertificateFile,
```

Comment:
I would definitely leave a comment for the next person to look into this, with some context around this :)

(and there is going to be a next person, at the latest when `tlsCertificateFile` starts going away in the driver entirely)

Reply:
Good idea yeah!

Reply:
Added some comment here at least about the fact that the option is not the same as the url param, there is more doc in the jsdoc of  `onnectionOptions.tlsCertificateFile` already.